common-functions

```diff
@@ -226,7 +231,7 @@ service_backup_auth() {
   declare SERVICE="$1" AWS_ACCESS_KEY_ID="$2" AWS_SECRET_ACCESS_KEY="$3" AWS_DEFAULT_REGION="$4" AWS_SIGNATURE_VERSION="$5" ENDPOINT_URL="$6"
   local SERVICE_BACKUP_ROOT="$PLUGIN_DATA_ROOT/$SERVICE/backup"
 
-  mkdir "$SERVICE_BACKUP_ROOT"
+  mkdir -p "$SERVICE_BACKUP_ROOT"
```

Comment:
Was this necessary?

Reply:
Otherwise it will fail upon trying to create the dir if it already existed